eg/servo-drive.js

```diff
@@ -26,11 +25,11 @@ board.on("ready", function() {
     invert: true // one wheel mounted inverted of the other
   });
 
-  wheels.both = new five.Servos().stop(); // reference both together
+  wheels.both = new Servos().stop(); // reference both together
```

Comment:
One thing I will never forgive myself for: allowing these to implicitly find all instances of any Servo. Can you update this to: 

```js
wheels.both = new Servos([wheels.left, wheels.right]).stop();
```

Reply:
Ya, that's weird. Done.